tools/coreml/converter/_layers.py

```diff
@@ -220,6 +220,39 @@ def convert_activation(net, node, module, builder):
                            output_name = output_name)
 
 
+def convert_leakyrelu(net, node, module, builder):
```

Comment:
Can you add a test case for this here: https://github.com/apache/incubator-mxnet/blob/master/tools/coreml/test/test_mxnet_converter.py?

If you can include tests for both leakyrely and prelu, that'll be great.

Reply:
Added in c94bbb8